docs/reference/release-notes/highlights.asciidoc

```diff
@@ -13,11 +13,30 @@ For detailed information about this release, see the <<es-release-notes>> and
 
 // Use the notable-highlights tag to mark entries that 
 // should be featured in the Stack Installation and Upgrade Guide:
+
 // tag::notable-highlights[] 
-// [discrete]
-// === Heading
-//
-// Description. 
+[discrete]
+=== Better space efficiency for `keyword`, `match_only_text`, and `text` fields
+
+The inverted index switched to a new encoding that is more space-efficient.
```

Comment:
Some users may not be familiar with internal indices or know they're an internal data structure.
```suggestion
We've updated inverted indices, an internal data structure, to use a more space-efficient encoding.
```